docs/contributing/index.md

```diff
@@ -1,5 +1,12 @@
 # Contributing
 
+## Contribution flow
+
+Did you find a software bug? -> Open an issue in the corresponding software repository
+Did you find strange behavior? -> [Ask in Q&A](https://github.com/autowarefoundation/autoware/discussions/categories/q-a)
+Do you have questions about Autoware functionality? -> [Ask in Q&A](https://github.com/autowarefoundation/autoware/discussions/categories/q-a)
+Do you want to add new features to Autoware? -> [Make a feature request](https://github.com/autowarefoundation/autoware/discussions/categories/feature-requests)
+
 This page explains how to contribute to Autoware.
```

Comment:
Please do not add text before this sentence. It's a description for the entire page.

Reply:
Yes. But the following part has duplication https://github.com/autowarefoundation/autoware/discussions/202 so I briefly added this

Reply:
I'm sorry, I couldn't understand what you mean. :thinking:

Reply:
As Lalith mentioned we need to change the page structure before adding the contents above so I temporarily added them here

Reply:
Okay, I understand. :smile: